FileProcessor/src/main/java/com/group7/FileProcessor/Main.java

```diff
@@ -199,6 +228,30 @@ public void run() {
      * @param path - File path to the .JSON file
      */
     private void processJSON(String path) {
-        System.out.println("TODO: Add the JSON reading code. Waiting on TA response for Jackson Library.");
+        //String to store the .JSON file into
+        StringBuilder sb = new StringBuilder();
+
+        //open a try catch to try and read in the file
+        try {
+
+            File jsonFile = new File(path);
+            Scanner scanner = new Scanner(jsonFile);
+
+            //Read in the .JSON file
+            while (scanner.hasNextLine()) {
+                //add in the line from the file
+                sb.append(scanner.nextLine());
+                sb.append("\n"); //add in a new line to prepare for the next itteration of the loop
+
+            }
+            
+            //Pass the file String to the Sequencer
+            //Sequencer.file(sb.toString());
```

Comment:
Right here is where it will link to the rest of the program. Either the sequencer or Senario.java. What ever you @Fluff-E think is best.